indico/htdocs/sass/partials/_boxes.scss

```diff
@@ -407,10 +403,42 @@ $i-box-padding: 10px;
 
     > li {
         @include border-top();
-        display: block;
-        padding: 10px 10px;
+        padding: 10px;
         position: relative;
         overflow: hidden;
+
+        /* Sortable row with handle on the left */
```

Comment:
Use a `//` comment - no need to include such comments in the final CSS (`//` are scss comments that are skipped when compiling it)